src/Brick/Widgets/List.hs

```diff
@@ -602,6 +603,31 @@ listFindBy test l =
         result = tailResult <|> headResult
     in maybe id (set listSelectedL . Just . fst) result l
 
+-- | Traversal that targets the selected element, if any.
+--
+-- Complexity: depends on usage as well as container type
+--
+-- @
+-- listSelectedElementL for 'List': O(1) -- preview, fold
+--                                O(n) -- set, modify, traverse
+-- listSelectedElementL for 'Seq.Seq': O(log(min(i, n - i)))  -- all operations
+-- @
+--
+listSelectedElementL
+  :: (Splittable t, Traversable t, Semigroup (t e))
+  => Traversal' (GenericList n t e) e
+listSelectedElementL f l =
+  case l ^. listSelectedL of
```

Comment:
I'll take care of this change in this case, but I wanted to mention for future PRs: it would help me if the style of the patches matches the style of the surrounding code, i.e. four-space indentation and `let` style as follows:
```haskell
let foo = ...
    bar = ...
in ...
```